### PR TITLE
docs: specify function return value

### PR DESCRIPTION
**Pull request by kingluo**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: CODE_STYLE.md [hunk 1]
```diff
@@ -320,6 +320,10 @@ local function check(age, name)
 end
 ```
 
+The function should return `<boolean>`, `err`.
```
Comment (spacewander) [inline CODE_STYLE.md:323]: Change in the doc should not begin with the prefix "fix"

Comment (membphis) [reply CODE_STYLE.md:323]: @kingluo any updates?

Comment (membphis) [reply CODE_STYLE.md:323]: we'd better to add some good or bad example, make it clearer

File: CODE_STYLE.md [hunk 2]
```diff
@@ -320,6 +320,24 @@ local function check(age, name)
...
+The first return value means successful or not, if not, the second return value specifies the error message.
+The error message can be ignore in some case.
```
Comment (tokers) [inline CODE_STYLE.md:325]: The error message can be ignored in some case.

**Review by membphis — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**